### PR TITLE
Replace resolve_message_type with roslib.message.get_message_class #49

### PR DESCRIPTION
**Pull request by yeliztaneroglu**

### CODE REVIEW

File: src/openag_brain/software_modules/topic_filter.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from couchdb import Server
 
 from openag_brain.util import resolve_message_type
```
Comment (gordonbrander) [inline src/openag_brain/software_modules/topic_filter.py]: We should remove resolve_message_type if no longer used anywhere.

File: src/openag_brain/software_modules/sensor_persistence.py [hunk 2]
```diff
@@ -27,6 +27,7 @@
 
 from openag_brain import params
 from openag_brain.util import resolve_message_type
```
Comment (gordonbrander) [inline src/openag_brain/software_modules/sensor_persistence.py]: We should remove resolve_message_type if not used. (would also be good to check that it is unused elsewhere in the codebase + remove from utils)

**Comment by yeliztaneroglu** [comment]
Okey, I removed resolve_message_type if not used. Thank you so much :+1:  @gordonbrander

File: src/openag_brain/software_modules/topic_connector.py [hunk 3]
```diff
@@ -25,6 +25,7 @@
 from openag_brain import params
 from openag_brain.srv import Empty
 from openag_brain.util import resolve_message_type
```
Comment (gordonbrander) [inline src/openag_brain/software_modules/topic_connector.py]: This import should be removed. We should also remove the `resolve_message_type` function definitition from `openag_brain.util` if it's no longer being used anywhere.

Thanks again for your contribs! Super excited to have you join the community. :)

**Comment by gordonbrander** [comment]
Going to test this out today. Assuming it works, with the small changes above it looks great!

File: src/openag_brain/software_modules/topic_connector.py (hunk 3 above)
Comment (yeliztaneroglu) [inline src/openag_brain/software_modules/topic_connector.py]: I am so excited, too : ) yaay \o/

**Comment by yeliztaneroglu** [comment]
I removed resolve_message_type in topic_connector.py, Thank you so much for your time :)

**Comment by yeliztaneroglu** [comment]
review ?  @gordonbrander

**Comment by gordonbrander** [comment]
Looking at it today :D

**Comment by yeliztaneroglu** [comment]
awesome :smile:

**Comment by gordonbrander** [comment]
Tested. It works.!

**Pull request merged**